app/controllers/api/v1/posts_controller.rb

```diff
@@ -10,7 +10,10 @@ def index
 
     api :GET, '/posts/:id', 'Show specific post'
     def show
-      success(@post)
+      @post.increment
```

Comment:
`increment` is a bit ambiguous. What about using `add_instance_counter_for :view` in the model and invoke `incr_view_count` here?